crates/sui-core/src/authority/epoch_start_configuration.rs

```diff
@@ -56,9 +56,9 @@ impl EpochStartConfiguration {
         ))
     }
 
-    pub fn new_for_testing() -> Self {
+    pub fn new_for_testing_with_rgp(rgp: u64) -> Self {
```

Comment:
Not quite sure if this is the right place to set it.
The most proper way seems to be setting the param in Genesis?

Reply:
Updated in the later commit.